app/models/megaphone/episode.rb

```diff
@@ -191,9 +212,31 @@ def audio_is_latest?
     end
 
     def set_enclosure
-      return unless audio_file_status == "success"
-      return if download_url.blank? || feeder_episode.enclosure_override_url.present?
-      feeder_episode.update(enclosure_override_url: download_url, enclosure_override_prefix: true)
+      return if download_url.blank? || !feeder_episode.complete_media?
+      set_external_media
+      feeder_episode.update!(enclosure_override_url: download_url, enclosure_override_prefix: true)
+    end
+
+    # if it's not published yet, can't get the file from mp, create as "complete"
+    def set_external_media
+      return if feeder_episode.published?
+
+      emr_attributes = {
+        status: :complete,
+        mime_type: "audio/mpeg",
+        medium: "audio",
+        original_url: download_url,
+        file_size: feeder_episode.media_file_size_sum,
+        duration: feeder_episode.media_duration_sum,
+        bit_rate: private_feed.audio_format[:b].to_i,
+        channels: private_feed.audio_format[:c].to_i,
+        sample_rate: private_feed.audio_format[:s].to_i
```

Comment:
I could also get these audio format attributes from megaphone, but I know they will be in DT, there may be timing issues getting them from mp.